.github/workflows/pull_request.yml

```diff
@@ -7,31 +7,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.13
+    - name: Set up Go 1.15
       uses: actions/setup-go@v1
       with:
-        go-version: 1.13
+        go-version: 1.15
       id: go
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
     - name: Get dependencies
+      env:
+        KUBEBUILDER_VER: "2.2.0"
       run: |
         go get -v -t -d ./...
         curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
         sudo mv kustomize /usr/local/bin/
         os=$(go env GOOS)
         arch=$(go env GOARCH)
-        curl -L https://go.kubebuilder.io/dl/2.2.0/${os}/${arch} | tar -xz -C /tmp/
-        sudo mv /tmp/kubebuilder_2.2.0_${os}_${arch} /usr/local/kubebuilder
+        curl -L "https://go.kubebuilder.io/dl/${KUBEBUILDER_VER}/${os}/${arch}" | tar -xz -C /tmp/
+        sudo mv "/tmp/kubebuilder_${KUBEBUILDER_VER}_${os}_${arch}" /usr/local/kubebuilder
         export PATH=$PATH:/usr/local/kubebuilder/bin
         curl https://www.foundationdb.org/downloads/6.2.20/ubuntu/installers/foundationdb-clients_6.2.20-1_amd64.deb -o fdb.deb
         sudo dpkg -i fdb.deb
 
     - name: Build
+      # Currently the default runner has 2 vCPU:
+      # https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources
+      env:
+        GOMAXPROCS: "2"
```

Comment:
It seems like setting `GOMAXPROCS` make the tests more reliable without the `connection refused` issues.